posthog-node/src/client.ts

```diff
@@ -639,6 +648,43 @@ export abstract class PostHogBackendClient extends PostHogCoreStateless implemen
     return super._shutdown(shutdownTimeoutMs)
   }
 
+  private async _requestRemoteConfigPayload(flagKey: string): Promise<PostHogFetchResponse | undefined> {
+    if (!this.options.personalApiKey) {
+      return undefined
+    }
+
+    const url = `${this.host}/api/projects/@current/feature_flags/${flagKey}/remote_config/`
+
+    const options: PostHogFetchOptions = {
+      method: 'GET',
+      headers: {
+        ...this.getCustomHeaders(),
+        'Content-Type': 'application/json',
+        Authorization: `Bearer ${this.options.personalApiKey}`,
+      },
+    }
+
+    let abortTimeout = null
+    if (this.options.requestTimeout && typeof this.options.requestTimeout === 'number') {
+      const controller = new AbortController()
+      abortTimeout = setTimeout(() => {
```

Comment:
In the old implementation this was `safeSetTimeout`. Is there a reason it was changed to `setTimeout` here? Seems like `safeSetTimeout` is the right choice.

```suggestion
      abortTimeout = safeSetTimeout(() => {
```